src/runtime/contrib/cudnn/conv_forward.cc

```diff
@@ -63,7 +64,8 @@ void ConvolutionForward(
   // Note: For 2D tenor, using ND setters causes CUDNN_STATUS_NOT_SUPPORTED error
   // in following cudnnGetConvolutionForwardWorkspaceSize() when data type is fp16, int
   if (dims == 2) {
-  // Set Desc
+    // Set Desc
+    CUDNN_CALL(cudnnSetConvolutionGroupCount(entry_ptr->conv_entry.conv_desc, groups));
```

Comment:
Since both if then and else have this, you should move this before the if/else.

Reply:
good point. Fixed.